featuretools/primitives/utils.py

```diff
@@ -40,6 +41,11 @@ def get_transform_primitives():
     return {prim.name.lower(): prim for prim in transform_primitives}
 
 
+def camel_and_title_to_snake(name):
+    name = re.sub('(.)([A-Z][a-z]+)', r'\1_\2', name)
+    return re.sub('([a-z0-9])([A-Z])', r'\1_\2', name).lower()
```

Comment:
I would expect "Top3Words" to be converted to "top_3_words" but currently this will return "top3_words"